core/api/transport/session.hpp

```diff
@@ -48,6 +48,8 @@ namespace kagome::server {
 
     Session(Socket socket, Id id, io_context &context, SessionManager &manager);
 
+    Session(Session &&other);
```

Comment:
you can delete copy and move constructors. why do you need them at all?

Reply:
clang-tidy issue :(

Reply:
just do = delete

Reply:
removed non-default destructor and move constructor. probably default destructor will be enough.